gradle-config.gradle

```diff
@@ -0,0 +1,22 @@
+//
+// Configuration file for gradle build execution.
+//
+
+task accessToken {
+  def tokenFile = new File("${projectDir}/src/main/res/values/developer-config.xml")
+  if (!tokenFile.exists()) {
+    String tokenFileContents = "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n" +
+      "<resources>\n" +
+      "    <string name=\"mapbox_access_token\">" + "$System.env.MAPBOX_ACCESS_TOKEN" + "</string>\n" +
+      "</resources>"
+
+    if (tokenFileContents == null) {
```

Comment:
@zugaldia you're completely right.
👀  I modified a bit the script [here](https://github.com/mapbox/mapbox-navigation-android/pull/11).
BTW, we should [get a consensus](https://github.com/mapbox/mapbox-gl-native/issues/8783#issuecomment-296622518) and have same script across all repos.
cc @cammace